frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -30,17 +31,18 @@ const OPENSHIFT_VERSION_QUERY = 'openshift_build_info{job="apiserver"}';
 
 const CAPACITY_MEMORY_TOTAL_QUERY = 'sum(kube_node_status_capacity_memory_bytes)';
 
-const CAPACITY_STORAGE_TOTAL_QUERY = 'sum(kubelet_volume_stats_capacity_bytes)'; // available with Ceph
-const CAPACITY_STORAGE_USED_QUERY = 'sum(kubelet_volume_stats_used_bytes)';
-const CAPACITY_STORAGE_TOTAL_DEFAULT_QUERY = 'sum(node_filesystem_avail_bytes)';
-const CAPACITY_STORAGE_USED_DEFAULT_QUERY = 'sum(node_filesystem_avail_bytes) - sum(node_filesystem_free_bytes)';
+const CAPACITY_STORAGE_TOTAL_QUERY = 'max_over_time(ceph_cluster_total_bytes[60m:])'; // available with Ceph
```

Comment:
I dont think this is correct - we want latest value of storage capacity, not max in last 60minutes. You can get wrong result if capacity goes down in the given timeframe.

Reply:
it should be just `ceph_cluster_total_bytes` and `sum(node_filesystem_avail_bytes)` IMO

Reply:
Actually this is different for Capacity and Utilization.
For Capacity, you are right - it shows the "actual". 
For utilization, we need to use the maximum of the time period as the last value can be lower than a former one.

I will fix that for the Capacity flow.

Reply:
But we are lucky, we do not need to deal with that explicitly as there is the search for maximal past value multiplied by 2. So we should be fine even with the chart.

Reply:
done